components/ArticleCard.jsx

```diff
@@ -1,17 +1,20 @@
-
-
 function ArticleCard(props) {
-    const {article} = props
-    return (
-      <div id="container">
+  const { article } = props;
+  const articleId = `/articles/${article.article_id}`
+
+  return (
+    <div id="container">
+      <a href={articleId}>
```

Comment:
No <a> tags in this project please as there is undesirable default effects associated with this tag. Please use a react <Link> instead